src/core/api/__tests__/public_api.test.ts

```diff
@@ -295,31 +300,14 @@ describe("API - Public API", () => {
         expect(player.isMute()).toBe(false);
       });
 
-      it("should return true in isMute if the volume is equal to 0", () => {
+      it("should not true in isMute if just the volume is equal to 0", () => {
```

Comment:
Well it could return true if it is indeed muted. That's why I also added the "just" in the sentence: isMute should not be influenced by volume settings.

Reply:
I think I didn't understand the sentence "should not true in isMute if just the volume is equal to 0"
it lack a verb or is it me not understanding ?

Reply:
done